cmd/rook/rook/rook.go

```diff
@@ -177,6 +180,24 @@ func GetOperatorImage(clientset kubernetes.Interface, containerName string) stri
 	return containerImage
 }
 
+func GetOperatorImagePullSecrets(clientset kubernetes.Interface) []v1.LocalObjectReference {
```

Comment:
I like the strategy here to use the operator's image pull secrets. I'm wondering though, is there any reason why the operator would have different image pull secrets from the Ceph cluster? If not, then it may be better to just use the operator's image pull secrets.

Reply:
I think there is no such case :) . But I added the option because image option exists. And where image appeares, image pull secrets appeares too.

Reply:
If there is no such case, I think I would prefer to have one and only one way of setting the image pull secrets, which I think makes sense to be done automatically based on the operator's currently configured image pull secrets.